Starting a new project.md

```diff
@@ -0,0 +1,68 @@
+## Starting a new project
+
+When you're starting a new project, there are a few things you need to do:
+
+#### 1. Open a repository
+
+Ask your team lead or DevOps to open a repository.
+
+#### 2. Fill out Project setup sheet
+
+We've come up with a project setup template in cooperation with DevOps team. It's purpose is to provide the DevOps with all the details needed for deploying your application to staging and production servers.
+
+It contains general info about your project that your project manager can fill out as well as more technical information like frameworks and technologies used, build and deploy setup, third party services that need to be set up...
+
+Ask your PM for a link to the sheet template.
+
+**Fill in the blanks on Backend related stuff.**
```

Comment:
Just one comment here: We're creating a new project setup template and it's going to have a separate sheet for backend related stuff.